scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -145,6 +144,8 @@
 	 */
 	private Long autoResumptionTimeoutMillis;
 
+	private boolean sniEnabled = true;
```

Comment:
With PR #336 we changed the design for this configuration to use references. 
This allows a upper layer to provide it's own defaults by checking, if the value was set, or a default should be used (that's the null case). I would prefer, if this `sniEnable` gets changed to `Boolean`.

Reply:
So, the idea would be to also add a getter for this property in the builder and then set the property to its default (true) in the *build()* method if it hasn't been set explicitly yet, right?

Reply:
yes.